encoders/image/CLIPImageEncoder/__init__.py

```diff
@@ -0,0 +1,48 @@
+__copyright__ = "Copyright (c) 2021 Jina AI Limited. All rights reserved."
+__license__ = "Apache-2.0"
+
+import numpy as np
+import torch
+
+from jina.executors.decorators import batching, as_ndarray
+from jina.executors.encoders.frameworks import BaseTorchEncoder
+from jina.executors.devices import TorchDevice
+
+class CLIPImageEncoder(BaseTorchEncoder):
+    """
+    Encodes data from a `np.ndarray` of shape `BatchSize x (Channel x Height x Width)` into
+    a `np.ndarray` of shape `Batchsize x EmbeddingDimension`.
+
+    Internally, :class:`CLIPImageEncoder` wraps the `CLIP` modeL from https://github.com/openai/CLIP
```

Comment:
The constructor params (model_name, *args, **kwargs) should be here, it will complain otherwise, could you document them in the class?

Reply:
all params have been added